src/psmove.c

```diff
@@ -505,13 +507,11 @@ psmove_count_connected_hidapi()
     while (cur_dev) {
 #ifdef _WIN32
         /**
-         * Windows Quirk: Ignore extraneous devices (each dev is enumerated
-         * 3 times, count only the one with "&col02#" in the path)
-         *
-         * We use col02 for enumeration, and col01 for connecting. We want to
-         * have col02 here, because after connecting to col01, it disappears.
+         * Windows Quirk: Each dev is enumerated 3 times.
+         * The one with "&col01#" in the path is the one we will get most of our data from. Only count this one.
+         * The one with "&col02#" in the path is the one we will get the bluetooth address from.
          **/
-        if (strstr(cur_dev->path, "&col02#") == NULL) {
+        if (strstr(cur_dev->path, "&col01#") == NULL) {
```

Comment:
The original comment confused me because no device was actually opened until much later, and at that point it was the device with col01 in the path that was opened. However, in this new version, I took the comment at its word and made sure to open the col02 device first (the one used for getting the BT address) and then open the col01 device.